hikey_debian.mk

```diff
@@ -226,7 +226,7 @@ PERF_FLAGS := CROSS_COMPILE_HOST=$(CROSS_COMPILE_NS_USER) \
 	TA_DEV_KIT_DIR=$(OPTEE_OS_TA_DEV_KIT_DIR)
 
 aes-perf: optee-os optee-client
-	$(MAKE) -C -j$(NPROC) $(AESPERF_PATH) $(PERF_FLAGS)
+	$(MAKE) -j$(NPROC) -C $(AESPERF_PATH) $(PERF_FLAGS)
```

Comment:
I can't see how that ever has worked. But aes-perf hasn't been called from the `all` target. In fact both aes-perf and sha-perf has been integrated into xtest. So I think it's better to remove everything that has to do with those two in this particular makefile and just use it from xtest.